ex7.12_NumPy_remove_single_dimensiona.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+
+def remove_single_dimensional(array_remove: np.array):
```

Comment:
What about return type? It is not clear when I call the function what I;m getting. Here you returned the shape for example.